AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/AWSCloudWatchLoggingSessionController.swift

```diff
@@ -145,10 +148,12 @@ final class AWSCloudWatchLoggingSessionController {
     }
 
     private func userIdentifierDidChange() {
-        resetCurrentLogs()
-        updateSession()
-        updateConsumer()
-        connectProducerAndConsumer()
+        Task {
```

Comment:
Same as above, this is now basically an async operation that we don't control.
From the method name it sounds it shouldn't really matter in this case, but just double checking.

Reply:
see above reply